governance/default.go

```diff
@@ -667,18 +667,25 @@ func (g *Governance) initializeCache() error {
 		}
 	}
 
-	lastGovernanceBlock, lastGovernanceStateBlock := indices[len(indices)-1], atomic.LoadUint64(&g.lastGovernanceStateBlock)
-	if lastGovernanceBlock >= lastGovernanceStateBlock {
-		// head block number is used to get the appropriate g.currentSet and g.actualGovernanceBlock
-		headBlockNumber := uint64(0)
-		if headBlockHash := g.db.ReadHeadBlockHash(); !common.EmptyHash(headBlockHash) {
-			if num := g.db.ReadHeaderNumber(headBlockHash); num != nil {
-				headBlockNumber = *num
-			}
+	// head block number is used to get the appropriate g.currentSet and g.actualGovernanceBlock
+	headBlockNumber := uint64(0)
+	if headBlockHash := g.db.ReadHeadBlockHash(); !common.EmptyHash(headBlockHash) {
+		if num := g.db.ReadHeaderNumber(headBlockHash); num != nil {
+			headBlockNumber = *num
+		}
+	}
+	newBlockNumber, newGovernanceSet, _ := g.ReadGovernance(headBlockNumber)
+	g.actualGovernanceBlock.Store(newBlockNumber)
+	g.currentSet.Import(newGovernanceSet)
```

Comment:
This code change `currentSet`, but doesn't call `g.updateGovernanceParams()`.

Reply:
thanks, reflected at https://github.com/klaytn/klaytn/pull/1168/commits/b59117a9563072e6f6772d2cbebb74bd800efc13.